packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -0,0 +1,143 @@
+import stringHash from 'string-hash';
+import idbQHelper from './idbHelper';
+import requestManager from './requestManager';
+import bcmanager from './broadcastManager';
+
+let _queue;
```

Comment:
I had a similar comment in my review. Switching to class-based member variables would be a lot cleaner in this instance.